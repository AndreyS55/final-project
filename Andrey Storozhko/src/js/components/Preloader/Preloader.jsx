import React from 'react';
import classNames from 'classnames';
import styles from './Preloader.scss';

const Preloader = (props) => (
  <div className={classNames(styles.preloader, {[styles.isLoading]: props.isLoading})}>
    <div className={styles.loading}>
      <div className={classNames([styles.pokeball, styles.normal])}></div>
      <div className={classNames([styles.pokeball, styles.great])}></div>
      <div className={classNames([styles.pokeball, styles.ultra])}></div>
      <div className={classNames([styles.pokeball, styles.master])}></div>
      <div className={classNames([styles.pokeball, styles.safari])}></div>
    </div>
  </div>
);

export default Preloader;