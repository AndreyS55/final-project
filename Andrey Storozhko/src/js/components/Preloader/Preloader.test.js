import React from 'react';
import Preloader from './Preloader';

it('should render a Preloader', () => {
  const wrapper = shallow(<Preloader />);
  expect(wrapper).toMatchSnapshot();
});
