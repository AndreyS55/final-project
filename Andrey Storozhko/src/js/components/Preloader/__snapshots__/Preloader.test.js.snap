// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Preloader 1`] = `
<div
  className="preloader"
>
  <div
    className="loading"
  >
    <div
      className="pokeball normal"
    />
    <div
      className="pokeball great"
    />
    <div
      className="pokeball ultra"
    />
    <div
      className="pokeball master"
    />
    <div
      className="pokeball safari"
    />
  </div>
</div>
`;
